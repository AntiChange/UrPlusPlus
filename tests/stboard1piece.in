3 8
*OOOXX*O
OOO*OOOO
*OOOXX*O
0 3
WWWSEEEEEEENW
2 3
WWWNEEEEEEESW
1
B 0 3
B 2 3
0
roll
a 2
roll
0 2
