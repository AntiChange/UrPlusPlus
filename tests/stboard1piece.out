Player 0's turn
Dice Roll: 4295032831, Flex Dice Roll:140728120237344
Board State:
*OOa  *O
OOO*OOOO
*OO1  *O
Player 0 game path:
v--<``E<
>------^
````````
Player 1 game path:
````````
>------v
^--<``E<
